### ISSUE TITLE
Learning Scheduler Issue

### ISSUE THREAD

**Issue by XingzhiZhou** [hiyouga/ChatGLM-Efficient-Tuning#393]
This work is amazing! I have a question regarding the learning scheduler issue. Even after setting lr_scheduler_type=linear, I noticed that the learning rate keeps decreasing with each step.

Here are a couple of examples:

{'loss': 0.6579, 'learning_rate': 0.0004952548330404218, 'epoch': 0.09}
{'loss': 0.6538, 'learning_rate': 0.000495079086115993, 'epoch': 0.1}

I would greatly appreciate it if you could kindly guide me on how to resolve this problem.

Thank you so much in advance!

**Comment by hiyouga** [comment]
"linear" means the learning rate will decrease in a constant speed.

**Comment by XingzhiZhou** [comment]
Thanks for your reply. I just realized I misunderstood "linear" to "constant".
